be/src/storage/rowset/segment_rewriter.cpp

```diff
@@ -23,7 +23,7 @@ SegmentRewriter::SegmentRewriter() = default;
 Status SegmentRewriter::rewrite(const std::string& src_path, const std::string& dest_path,
                                 const std::shared_ptr<const TabletSchema>& tschema, std::vector<uint32_t>& column_ids,
                                 std::vector<std::unique_ptr<Column>>& columns, uint32_t segment_id,
-                                const FooterPointerPB& partial_rowset_footer) {
+                                const FooterPointerPB& partial_rowset_footer, uint64_t* file_size) {
```

Comment:
IMO, change the return type to `StatusOr<int64_t>` is more clear than adding a new output parameter.